configen/OptionsParser.swift

```diff
@@ -71,7 +69,7 @@ final class OptionsParser {
       let (variableName, type) = (hints[0], hints[1])
       hintsDictionary[variableName] = type
     }
-
-    return hintsDictionary
+    
+    return hintsDictionary.sorted(by: { $0.0 < $1.0 })
```

Comment:
You can write this in point-free style: `hintsDictionary.sorted(by: <)`

Reply:
Nice. Updated